CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.0.0-alpha.0]
+### Added
+
+### Changed
+
+### Removed
+
+
+[Unreleased]: https://github.com/ethereum/web3.js/compare/v1.0.0-beta.55...HEAD
```

Comment:
is this going to be `2.0.0-alpha.0` ? as if so any changes in `beta-55` should go into the tabs of `2.0.0-alpha.0` i would say. And then kill the `[Unreleased]: https://github.com/ethereum/web3.js/compare/v1.0.0-beta.55...HEAD` because that wont be a thing anymore.

Reply:
The unreleased tag is from the referenced [changelog standard](https://keepachangelog.com/en/1.0.0/#effort) and should show the state between a release. The added URL can be helpful for having an overview of all the happened changes since the last release.